src/app/personside/infotabs/oversikt/__snapshots__/Oversikt.test.tsx.snap

```diff
@@ -459,9 +459,9 @@ exports[`Viser oversikt med alt innhold 1`] = `
                 <p
                   className="typo-normal"
                 >
-                  20.05.2015
+                  21.05.2015
```

Comment:
Her virker det som at mock-dataen ikke er statisk. Denne kommer nok til å brekke en gang om dagen ;-) Det går an å mock Date.now og Date.getTime for å motvirke dette, er gjort i en del tester

Reply:
Yes. Ser dette problemet kommer på hver commit jeg gjør.